pkg/reconciler/cronjobsource/cronjobsource.go

```diff
@@ -75,6 +73,10 @@ type Reconciler struct {
 // Check that our Reconciler implements controller.Reconciler
 var _ controller.Reconciler = (*Reconciler)(nil)
 
+type env struct {
```

Comment:
IMO the `env` struct should be defined in `controller.go`. Defining it here means knowledge of envconfig is spread across two files when really it just needs to be in one.

Reply:
Done.

Reply:
I have used this pattern and I like it a lot: https://github.com/GoogleCloudPlatform/cloud-run-events/blob/master/pkg/pubsub/adapter/adapter.go#L36

Reply:
and then in use: https://github.com/GoogleCloudPlatform/cloud-run-events/blob/master/cmd/pubsub/receive_adapter/main.go#L40-L58